spec/features/log_as_admin_and_create_admin_user.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+feature "Successful login" do
+  before :each do
+    @user = AdminUser.create!(email: "admin@example.com", password: "password", password_confirmation: "password")
+  end
+
+  background do
+    visit "/admin"
+  end
+
+  scenario "Title has login" do
+    expect(page).to have_content("Login")
+  end
+
+  scenario "Check if we are at the login page" do
+    expect(page).to have_css("div", text: "You need to sign in or sign up before continuing.")
+  end
+
+  scenario "Confirm that have: remember me" do
+    within("form#session_new") do
+      expect(page).to have_content("Remember me")
+    end
+  end
+
+  scenario "do we have right current path" do
+    expect(page).to have_current_path("/admin/login")
```

Comment:
I would merge those scenarios into one. Each scenario has performance overhead for setting everything up. It will be executed faster in one spec